ovve_ui/utils/in_packet.py

```diff
@@ -91,17 +91,29 @@ def calculate_runstate(self, mode_value):
         
         startBit = None
 
+        # if mode_value == 0:
+        #     startBit = 0
+        # elif mode_value == 1:
+        #     startBit = 1
+        # elif mode_value == 2:
+        #     startBit = 0
+        # elif mode_value == 3:
+        #     startBit == 1
+        # elif mode_value == 4:
+        #     startBit == 0
+        # elif mode_value == 5:
+        #     startBit = 1
         if mode_value == 0:
-            startBit = 0
-        elif mode_value == 1:
+                startBit = 0
```

Comment:
This can be done via bitwise operations much cleaner:

    startBit = ((mode_value >> 7) & 0x1)
    ventilationMode = (mode_value & 0x7f)